src/CattyTests/Views/Data/BrickCellVariableDataTests.swift

```diff
@@ -0,0 +1,164 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+import XCTest
+
+@testable import Pocket_Code
+
+class BrickCellVariableDataTests: XCTestCase {
+
+    func testInitWithFrameWhenObjectVariableInContainer() {
+
+        let spriteObject = SpriteObject()
+        spriteObject.name = "testObject"
+
+        let variablesContainer = VariablesContainer()
+        let userVariable1 = UserVariable(name: "testVariable1")
```

Comment:
I would suggest to rename these variables to `objectVariable1`, `objectVariable2`, `projectVariable` which makes it more readable because you directly know which variable is available at which scope.